x-pack/legacy/plugins/code/public/components/commits/commit.tsx

```diff
@@ -55,7 +54,7 @@ const revisionLinkLabel = i18n.translate('xpack.code.commits.revisionLinkAriaLab
 });
 
 const CommitActions = ({ commitId, repoUri }: ActionProps) => {
-  const revisionPath = `#/${repoUri}/${PathTypes.tree}/${commitId}`;
+  const revisionPath = `#/${repoUri}/commit/${commitId}`;
```

Comment:
Should we add a `PathType` for this new path, or is that stuff going away? I know we've been alluding to some more robust path-generation helpers, but I'm not sure where we landed.

Reply:
`commit` is not one of the `PathTypes`

Reply:
`PathTypes` defines different path types in source view page whose URIs share the same structure as `/$repoUri/.../$filePath`, while diff page is a different page.

Reply:
Got it, thanks!